client/src/Login/Login.js

```diff
@@ -52,21 +53,23 @@ export function Login() {
     })
     if (!response.ok || response.headers.get('authorization') === null) {
       const errorMessage = await response.json()
+      dispatch(removeAuth())
       setApiError({
         status: response.status,
         message: errorMessage.error,
         attribute: errorMessage.attribute,
         type: errorMessage.type,
         context: { email: values.email }
       })
+      history.push('/login')
```

Comment:
We don't need this since the user is already on the login page.

Reply:
Whoops good catch